app/services/sendspin_service.py

```
@@ -0,0 +1,836 @@
+            self._audio_thread.join(timeout=3)
+            self._audio_thread = None
+
+    def _terminate_active_process(self) -> None:
+        with self._process_lock:
+            proc = self._active_process
+        if proc:
+            try:
+                proc.terminate()
+                proc.wait(timeout=1)
+            except Exception:
+                pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)